languages/rust/.gitignore

```
@@ -0,0 +1,4 @@
+.DS_Store
+**/target
+**/exercises/**/*/Cargo.lock
```

`**/*` is semantically redundant and may be computationally expensive, depending on the expansion strategy.

```suggestion
**/exercises/**/Cargo.lock
```